### ISSUE TITLE
Request to expose tasks append method

### ISSUE THREAD

**Issue by onazaruk** [pgjones/quart-tasks#3]
Hi,

Wouldn't it be a good idea to expose a method to append tasks to the _tasks list?
**Usecase**:
I want the periodic background tasks to start only after some startup logic is done.

So currently I am calling the following after my startup logic is finished:
```python
        tasks._tasks.append(quart_tasks.extension._PeriodicTask(
            timedelta(minutes=5), None, refresh))
```

It would be neat to have `tasks.append()`, `tasks.delete()` methods for easier usage!

Thanks!

**Comment by pgjones** [comment]
Yep I think that is a good idea, 528638b0deb72134bcad252293be81941d71801c. Not sure about deletion though
